app/models/organization_membership.rb

```diff
@@ -23,4 +25,10 @@ class OrganizationMembership < ApplicationRecord
   def update_user_organization_info_updated_at
     user.touch(:organization_info_updated_at)
   end
+
+  private
+
+  def bust_cache
+    Organizations::BustCacheWorker.perform_async(organization.id, organization.slug)
```

Comment:
Hmmm, this busts cache for all the organization's articles as well but that's probably okay?

Reply:
That's a good point @filleduchaos . 

I think busting all org's articles on new member is a bit overkill. I think a good alternative is `BustCachePathWorker.perform_async(organization.path.to_s)`

Reply:
Agreed @maestromac

Reply:
Thanks for the suggestion.

I have updated the code and test accoridngly.

cc: @filleduchaos @maestromac @benhalpern